docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,135 @@
+# Release 369 (19 Jan 2022)
+
+## General
+
+* Add support for `Pacific/Kanton` time zone. ({issue}`10679`)
+* Display `Physical input read time` using most succinct time unit in `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`10576`)
+* Fine tune request retry mechanism in HTTP event listener. ({issue}`10566`)
+* Add support for using PostgreSQL and Oracle with database backed resource
+  groups. ({issue}`9812`)
+* Remove unnecessary fine-grained spill toggles. ({issue}`10591`)
+* Use formatting specified in the SQL standard when casting `double` and `real`
+  values to `varchar` type. ({issue}`552`)
+* Add experimental implementation of task level retries. ({issue}`9818`)
+* Prevent hanging query execution on failures with `phased` execution policy.
+  ({issue}`10656`)
+* Fix `UnsupportedOperationException` in `max_by` and `min_by` aggregation.
+  ({issue}`10599`)
+* Fix error message for function calls with lambda arguments. ({issue}`10423`)
+* Fix incorrect results or failure when casting date to bounded `varchar` type.
```

Comment:
bounded `varchar` -> `varchar(n)`. 

There's no such a thing as "bounded varchar". `varchar(n)` and `varchar` are two distinct types (with an unfortunate common name)